shuup/admin/templates/shuup/admin/auth/login.jinja

```diff
@@ -40,10 +40,17 @@
 {% endblock %}
 {% block extra_js %}
 <script>
+
 $(function () {
     var fields = $("input");
 
     fields.each(function() {
+        if ($(this).val().length > 0) {
```

Comment:
Does this work now on cases where you have automatic field filler (for example password manager)?